src/app/components/team/TeamReport.js

```diff
@@ -85,88 +86,98 @@ class TeamReportComponent extends React.Component {
           onChange={this.handleChangeLanguage.bind(this)}
         />
         <Can permissions={team.permissions} permission="update Team">
-          <p style={{ marginTop: units(2), textAlign: 'end' }}>
-            <Button onClick={this.handleSubmit.bind(this)} primary>
-              <FormattedMessage id="teamReport.save" defaultMessage="Save" />
-            </Button>
-          </p>
+          <Box mt={2} display='flex' alignItems='end' justifyContent='end'>
```

Comment:
Please use double quotes for string props/attributes.

`89:31  error  Unexpected usage of singlequote    jsx-quotes`